### DISCUSSION TITLE
Extend API documentations with implemented equations

### DISCUSSION THREAD

**Discussion by j042** [tataratat/splinepy#132] [Ideas]
Thanks to @jzwar, `Mapper` class has an awesome [documentation](https://tataratat.github.io/splinepy/splinepy.helpme.html#splinepy.helpme.mapper.Mapper). It seems that sphinx works quite well with writing equations and it'd great to have some more of those.   
Following classes don't have exact descriptions in forms of equations that we commonly see:
- [Spline](https://tataratat.github.io/splinepy/splinepy.html#splinepy.spline.Spline)
- [BSplineBase](https://tataratat.github.io/splinepy/splinepy.html#splinepy.bspline.BSplineBase)
- [BSpline](https://tataratat.github.io/splinepy/splinepy.html#splinepy.bspline.BSpline)
- [NURBS](https://tataratat.github.io/splinepy/splinepy.html#splinepy.nurbs.NURBS)
- [BezierBase](https://tataratat.github.io/splinepy/splinepy.html#splinepy.bezier.Bezier)
- [Bezier](https://tataratat.github.io/splinepy/splinepy.html#splinepy.bezier.Bezier)
- [RationalBezier](https://tataratat.github.io/splinepy/splinepy.html#splinepy.rational_bezier.RationalBezier)

A lot of member functions are documented / implemented in `Spline` and base classes, so maybe we can just write there for all possible cases for now. We will extend docs (besides API Ref) to include some general spline information and we can also clarify once more there.

**Comment by j042** [comment]
@danielwolff1 would you like to help us with this?

**Reply by danielwolff1** [reply]
Sure @j042, I'll happy to help out here and take care of it! 😁 👍

**Reply by j042** [reply]
thank you :D
